src/mdx-pages/guides/videojs-7-to-8.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: Video.js 7 to 8 Migration
+category: migration
+order: 1
+description: Migrating your usage of Video.js from 7 to 8
+---
+
+This guide describes backward incompatible changes made between Video.js 7 and Video.js 8 and how to migrate your implementation, if needed.
+
+## IE11 Support
+
+Video.js 8 removes support for IE11. Attempting to use Video.js 8 in IE11 will cause errors to be thrown. There is no migration path here; so, if you must have IE11 support, use an older version of Video.js.
+
+## Removals
+### `videojs.extend()`
+
+The `videojs.extend()` function is removed from Video.js 8.
+
+Video.js now uses native ES6 classes internally and the `extend()` function only worked with plain function prototypes. It did not work with ES6 classes, so it has been removed.
+
+At the most basic level, the old way to create a component using `extend()` was:
+
+```js
+const Component = videojs.getComponent('Component');
+
+const MyComponent = videojs.extend(Component, {
+  constructor: function(player, options) {
+    Component.call(this, player, options);
+  }
+});
+
+videojs.registerComponent('MyComponent', MyComponent);
+```
+
+Going forward, only ES6 classes are supported. The equivalent would be:
+
+```js
+const Component = videojs.getComponent('Component');
+
+class MyComponent extends Component {
+  constructor(player, options) {
+    super(player, options);
+  }
+}
+
+videojs.registerComponent('MyComponent', MyComponent);
+```
+
+### `firstplay` Event
+
+The `firstplay` event was removed from Video.js 8.
+
+Instead of listening for the `firstplay` event, use `one()` to bind a callback to the first occurrence of the `play` event, like this:
+
+```js
+player.ready(() => {
+  player.one('play', callback);
+});
+```
+
+### Setting Some Attributes as Properties
+
+In previous versions of Video.js, you could call the various flavors of `createEl()` with both DOM properties and attribute names in its second argument. While there is a one-to-one mapping of properties to attributes in most cases, there were a few attributes that we still allowed to be set via the second argument to this function:
```

Comment:
Do we warn/error about these not being set if they are passed in?

Reply:
They'll throw an error, I believe. Would have to check.

Reply:
No, they'll just be set as properties of the element and not attributes (meaning they'll be non-functional).

Reply:
Clarified this whole section.